### ISSUE TITLE
should we use readthedocs to host the tutorial

### ISSUE THREAD

**Issue by berndbischl** [mlr-org/mlr-tutorial#34]
see here
https://readthedocs.org/

this is based on md files I think

This uses this. It even has a  github badge.
https://github.com/aydindemircioglu/SVMBridge

@mllg @jakob-r @larskotthoff @zmjones

**Comment by berndbischl** [comment]
@schiffner

**Comment by jakob-r** [comment]
I don't want to. We are big enough to have our own tutorial page :wink: and I favor strongly to have full control over css and html to be able to format our learner tables in a good way.
What would be the pros for readthedocs?

**Comment by berndbischl** [comment]
I thought it might reduce the technical burden on our side. If you guys think it will only complicate things, lets not do it. 
@schiffner 
Do you want to close this?

**Comment by schiffner** [comment]
I checked this out two months ago, but totally forgot to reply.

Short answer: I agree with Jakob that it would complicate things.

Long answer:
Pros:
- Versioning is as far as I understand a little simpler as what we do now: Works by pushing tags
- It's comfortable: has automatic GitHub integration, Auto updating upon push ...
- the badge :)

Cons:
- The build chain gets longer: First let Travis knit the md-files, then let readthedocs build the html-files from the md-files
- Theme: With mkdocs one has to use the readthedocs theme. I tried this out locally. I like ours much better. One would again need to invest some work into the learner and measure tables.
- Many nice-to-have things like pdfs are only available with sphinx, not with mkdocs.
